doc/sources/conf.py

```diff
@@ -34,17 +36,16 @@
 
 sys.path.insert(0, os.path.abspath("../"))
 
-
 # -- Project information -----------------------------------------------------
 
 project = "Intel(R) Extension for Scikit-learn*"
 copyright = "Intel"
 author = "Intel"
 
 # The short X.Y version
-version = "2025.0.0"
+version = os.environ.get('DOC_VERSION','')
```

Comment:
How about changing the version to "latest" here when it doesn't find the env. variable?